query-planner-js/src/QueryPlan.ts

```diff
@@ -45,6 +51,10 @@ export interface FetchNode {
   outputRewrites?: FetchDataOutputRewrite[];
 }
 
+export interface SubscriptionNode extends Omit<FetchNode, 'kind' | 'hasDefers'> {
```

Comment:
I think you should also omit the field `id` because it's related to defer. I'm wondering if we should handle the `inputRewrites` and `outputRewrites` for a subscription ?